src/GEM_u8g2.cpp

```diff
@@ -167,7 +167,9 @@ void GEM_u8g2::init() {
 
   _u8g2.firstPage();
   do {
-    _u8g2.drawXBMP((_u8g2.getDisplayWidth() - _splash.width) / 2, (_u8g2.getDisplayHeight() - _splash.height) / 2, _splash.width, _splash.height, _splash.image);
+    //Fix for our non centered screen
+    //_u8g2.drawXBMP((_u8g2.getDisplayWidth() - _splash.width) / 2, (_u8g2.getDisplayHeight() - _splash.height) / 2, _splash.width, _splash.height, _splash.image);
+    _u8g2.drawXBMP((_u8g2.getDisplayWidth() - _splash.width) / 2, (96 - _splash.height) / 2, _splash.width, _splash.height, _splash.image);
```

Comment:
Puhh.. If we ever need a bigger screen, this could be hard to find.

Can we use any KeyWord in the comment that we could add to any readme in our LoGas Repo?

Reply:
I added a "TODO" keyword